addons/base_import_module/static/src/base_import_list_renderer.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module */
+import { useService } from "@web/core/utils/hooks";
+import { ListRenderer } from "@web/views/list/list_renderer";
+
+export class ImportListRenderer extends ListRenderer {
+
+    setup() {
+        super.setup()
+        this.action = useService("action");
+    }
+
+    get hasSelectors() {
+        return super.hasSelectors && this.props.list.records[0]?._values.module_type != 'industries';
+    }
+
+    async onCellClicked(record, column, ev) {
+        if (record._values.module_type && record._values.module_type !== 'official') {
+            const re_action = {
+                name: "more_info",
+                res_model: "ir.module.module",
+                type: "ir.actions.act_window",
+                views: [[false, "form"]],
+                context: {
+                    'module_name': record._values.name,
+                    'module_type': record._values.module_type,
+                }
+            }
+            this.action.doAction(re_action)
```

Comment:
can't we call this.env.services.action.doAction() as it is done in 'editGroupRecord' and avoid the overriding of setup() ?